lib/fakeroot.py

```diff
@@ -147,15 +147,18 @@
    # 1. CentOS seems to have only fakeroot, which is in EPEL, not the standard
    #    repos.
    #
-   # 2. Enabling EPEL can have undesirable side effects, e.g. different
+   # 2. Unlike on CentOS, RHEL doesn't have the epel-release rpm in the
+   #    standard repos, so install the rpm directly.
```

Comment:
Should clarify we're doing the RPM way for both. Or did you mean “unlike on Fedora”?

Reply:
> Good work. Some comments inline.
> 
> Were you planning to add the Debian 11 (#1043) and openSUSE (#1044) from PR #1035 here?
> 
> See also #1042.

I was going to make an issue for Debian 11 (doesn't appear out yet) and OpenSUSE, looks like you already did which is quite helpful!

Reply:
Will clarify that we install via RPM for both.